fcrepo-import-export-driver/src/test/java/org/fcrepo/importexport/integration/ExecutableJarIT.java

```diff
@@ -89,20 +92,67 @@ public void testJarExport() throws Exception {
         assertEquals(url, response.getLocation());
 
         // Run an export process
-        final ProcessBuilder builder = new ProcessBuilder("java", "-jar", EXECUTABLE,
-                "-m", "export",
+        final Process process = startJarProcess("-m", "export",
                 "-d", TARGET_DIR,
                 "-r", url.toString());
 
-        builder.directory(new File(TARGET_DIR));
-        final Process process = builder.start();
-        logger.debug("Output of jar run: {}", IOUtils.toString(process.getInputStream()));
+        // Verify
+        assertTrue("Process did not exit before timeout!", process.waitFor(TIMEOUT_SECONDS, TimeUnit.SECONDS));
+        assertEquals("Did not exit with success status!", 0, process.exitValue());
+
+        assertTrue(new File(TARGET_DIR, url.getPath() + ArgParser.DEFAULT_RDF_EXT).exists());
+    }
+
+    @Test
+    public void testExportBinaryAndRDFToSamePath() throws Exception {
+        final String content = "Hello World 😀";
+
+        // create a binary resource
+        final FcrepoResponse response = createUTF8PlaintextBinary(content);
+        assertEquals(CREATED.getStatusCode(), response.getStatusCode());
+        assertEquals(url, response.getLocation());
+
+
+        // Run an export process
+        final Process process = startJarProcess("-m", "export",
+                "-d", TARGET_DIR,
+                "-b", TARGET_DIR,
+                "-r", url.toString());
 
         // Verify
         assertTrue("Process did not exit before timeout!", process.waitFor(1000, TimeUnit.SECONDS));
         assertEquals("Did not exit with success status!", 0, process.exitValue());
 
-        assertTrue(new File(TARGET_DIR, url.getPath() + ArgParser.DEFAULT_RDF_EXT).exists());
+        response.getLinkHeaders("describedby")
```

Comment:
We need to also assert that a `describedby` Link header exists.
If I replace `describedby` with `junk` in this test, it still passes.